internal/output/release/release.go

```diff
@@ -82,7 +82,7 @@ func (o *Output) Permissions(filename string) os.FileMode {
 }
 
 func (o *Output) releaseScript(w io.Writer) error {
-	if _, err := w.Write([]byte("#!/bin/bash\n\n")); err != nil {
```

Comment:
lgtm, but you need to build kres and run it for this repo as well:

```
make _out/kres-linux-amd64
_out/kres-linux-amd64 gen
```

Reply:
I did this, but it didn't seem to regenerate the hack/release.sh file with the updated shebang. Am I missing another step?

Reply:
Ah silly me, that would require a make rekres :D

Reply:
Hmmmm, `make rekres` doesn't regenerate `hack/release.sh`

Reply:
I don't remember the rules, probably it doesn't because it ignores the preamble actually.

Reply:
`make rekres` pulls from ghcr.io, so it won't work on a PR'ed version.

not a big deal either way, but just keep in mind it won't update existing scripts

(sorry, Nix users :)

Reply:
aaaahhh, only regenerates it if it doesn't exist. Is this expected?